plugin/evm/config/config.go

```diff
@@ -379,25 +256,11 @@ func (c *Config) Validate(networkID uint32) error {
 	return nil
 }
 
-func (c *Config) Deprecate() string {
+// deprecate returns a string of deprecation messages for the config.
+// This is used to log a message when the config is loaded and contains deprecated flags.
+// This function should be kept as a placeholder even if it is empty.
```

Comment:
Out of curiosity why should this be kept as a placeholder even if empty?

Reply:
so that we can deprecate in case we need it again. Otherwise we need to reimplement the whole thing.